### ISSUE TITLE
Publication of cube metadata - Error message

### ISSUE THREAD

**Issue by ortnever** [zazuko/cube-creator#1227]
When I want to modify the metadata of the cube "ubd0066_Import de PROD" on Cube Creator INT
I get the following error message:
![image](https://user-images.githubusercontent.com/79834090/169777465-d74619f8-e6e8-440d-8d6d-23b4d389a5c3.png)

**Comment by l00mi** [comment]
I just fixed it.. please try again.

**Comment by ortnever** [comment]
It works. Thank you !

**Comment by ortnever** [comment]
same problem with
![image](https://user-images.githubusercontent.com/79834090/169783352-3ad86133-95df-486d-84a1-7c6ef9394d37.png)
![image](https://user-images.githubusercontent.com/79834090/169785458-964c579a-f49a-4d1d-80e9-e86eaf9a307a.png)
![image](https://user-images.githubusercontent.com/79834090/169785723-07eea9dc-60ff-4167-adeb-e1970a501a3c.png)

**Comment by l00mi** [comment]
@ortnever done.

**Comment by l00mi** [comment]
@tpluscode 

Btw. i also found this, imported from PROD:

```
	<https://schema.ld.admin.ch/contactPoint> [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	], [
		rdf:type schema:Person ;
		schema:name "Abteilung Lärm und NIS" ;
		schema:email "noise@bafu.admin.ch" ;
	] ;
```

**Comment by ortnever** [comment]
no, they are all "old" projects

**Comment by tpluscode** [comment]
I'm sorry. I will see again why this is still happening on import.

To mitigate is to remove the `http://purl.org/dc/terms/identifier` property in the RDF view.

<img width="620" alt="image" src="https://user-images.githubusercontent.com/588128/176851404-5b433b6d-eab4-47a9-8690-8cf36c05bc44.png">

**Comment by tpluscode** [comment]
I looked at the export and, understandably, if a project's metadata does indeed have a `dcterms:identifier`, it too will be included in the export and subsequently imported into a new project.

Otherwise, attempting to save and publish such cube will fail, which is the observed result, and by-design. 

However, to prevent importing a project like that, I have just submitted a change which adds additional validation of the imported file.
